common/djangoapps/third_party_auth/models.py

```diff
@@ -70,12 +70,21 @@ class ProviderConfig(ConfigurationModel):
     Abstract Base Class for configuring a third_party_auth provider
     """
     icon_class = models.CharField(
-        max_length=50, default='fa-sign-in',
+        max_length=50,
+        blank=True,
+        default='fa-sign-in',
         help_text=(
             'The Font Awesome (or custom) icon class to use on the login button for this provider. '
             'Examples: fa-google-plus, fa-facebook, fa-linkedin, fa-sign-in, fa-university'
         ),
     )
+    icon_image = models.ImageField(
+        blank=True,
+        help_text=_(
```

Comment:
So far we have not been translating these admin strings, because they're only seen by DevOps, and it would create extra work for all the translators with little benefit.